styles/cart.css

```diff
@@ -0,0 +1,64 @@
+table,
+th {
+  text-align: center;
+  border-bottom: 1px solid var(--gray-300);
+}
+th,
+tr,
+td {
+  padding: 0.5rem;
+}
+
+.cart-page {
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+.shop-link {
+  margin-top: 1rem;
+}
+
+.horizontal {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.hori-stats {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  gap: 1rem;
+}
+.my-cart {
+  margin-top: 1.5rem;
+  display: flex;
+}
+
+.qty-input {
+  padding: 0.5rem;
+  width: 3.5rem;
+  border: 1px solid var(--gray-500);
+  outline: none;
+  background-color: var(--gray-100);
+}
+.ct-product-stats h4 {
+  color: var(--gray-700);
+}
+
+.checkout {
+  text-align: end;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: start;
+  gap: 1rem;
```

Comment:
Just a suggestion : I have seem gap 1rem many times in ur code , maybe u can abstract out those as well? like create some gap classes gap-s { gap:1rem} gap-xs { gap: 0.5rem} and use these  classes as needed so u don't have to write each time?U can do it for margin and padding as well if u are using same quantity repeatedly, u can do this for next project I guess don't change it now as there would be too much refactoring

Reply:
Yes, I will going to refactor all flex properties in diff class and will do for gap too. Thank you for sugg.